lib/osf-components/addon/components/registries/review-form-renderer/template.hbs

```diff
@@ -1,11 +1,16 @@
-{{#each this.schemaBlockGroups as |group|}}
-    <Registries::SchemaBlockGroupRenderer
-        @schemaBlockGroup={{group}}
-        @renderStrategy={{
-            component
-            'registries/schema-block-renderer/read-only/mapper'
-            registrationResponses=@registrationResponses
-            node=@node
-        }}
-    />
+{{assert 'Registries::ReviewFormRenderer requires a draftManager' @draftManager}}
```

Comment:
💯

Reply:
I love being able to use template-only components when we can, but wanted a way to catch missing required args in development. It seems like an assert helper should be built into Ember. Maybe I'll open an RFC!